packages/admin/admin-theme/src/componentsTheme/MuiDataGrid.tsx

```diff
@@ -89,16 +93,37 @@ export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component, {
         },
         panelContent: {
             [`& .${gridClasses.filterForm}:first-child .${gridClasses.filterFormLinkOperatorInput}`]: {
+                display: "flex",
                 ["@media (max-width: 900px)"]: {
                     display: "none",
                 },
             },
         },
         filterForm: {
-            padding: spacing(4),
-
+            margin: spacing(5, 4, 0, 4),
+            padding: spacing(2, 1),
+            gap: "4px",
+            borderBottom: `1px solid ${palette.grey[50]}`,
+            "&:last-child": {
+                border: "none",
+                marginBottom: spacing(2),
+            },
+            "&:first-child": {
```

Comment:
Please avoid using `:first-child` as this causes the following error message in the console: 

```
The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".
```

```suggestion
            "&:first-of-type": {
```